integration/simple-cache-client.test.ts

```diff
@@ -19,24 +25,31 @@ const deleteCacheIfExists = async (
   momento: SimpleCacheClient,
   cacheName: string
 ) => {
-  try {
-    await momento.deleteCache(cacheName);
-  } catch (e) {
-    if (!(e instanceof NotFoundError)) {
-      throw e;
+  const deleteResponse = await momento.deleteCache(cacheName);
+  if (deleteResponse instanceof DeleteCache.Error) {
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-call
+    if (deleteResponse.errorCode() !== MomentoErrorCode.NOT_FOUND_ERROR) {
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-call
```

Comment:
why do we need this eslint disabled line on 32 here but not on 43  or 51?

Reply:
O I guess this is test but just want to double check make sure nothing weird happening here vs other API calls